roles/glusterfs/handlers/main.yml

```diff
@@ -0,0 +1,10 @@
+---
+- name: Restart mesos services
+  service: name={{item}} state=restarted
+  with_items:
```

Comment:
not only, but every service, writing to /mnt should be restarted, when /mnt being changed